README.md

```diff
@@ -51,7 +51,7 @@ You can also pass options for the REST API:x
       // fields have a cursor filter available
       safeFilters: [ 'slug' ],
       // Restrict GET routes to users with correct permission (false by default)
-      addPermissionToGetRoutes: true
+      getRequiredEditPermission: true
```

Comment:
This should be `getRequiresEditPermission`, present tense

Reply:
Ok, updated as requested